onboard/src/pages/subsections/backgroundInformation/EmploymentDesired.jsx

```diff
@@ -1,7 +1,253 @@
-import React from "react";
+import React, { useState } from "react";
+import Radio from "@mui/material/Radio";
+import RadioGroup from "@mui/material/RadioGroup";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import FormControl from "@mui/material/FormControl";
+import FormLabel from "@mui/material/FormLabel";
+import Grid from "@mui/material/Grid";
+import InputField from "../../../components/inputs/InputField";
+import Box from "@mui/material/Box";
+import { FormHelperText } from "@mui/material";
 
 function EmploymentDesired() {
-  return <></>;
+  const [position, setPosition] = useState("");
+  const [startDate, setStartDate] = useState("");
+  const [desiredSalary, setDesiredSalary] = useState("");
+  const [employmentStatus, setEmploymentStatus] = useState("");
+  const [contactCurrentEmployer, setContactCurrentEmployer] = useState("");
+  const [employedAtOCI, setEmployedAtOCI] = useState("");
+  const [nonCompeteStatus, setNonCompeteStatus] = useState("");
+  const [expirationDate, setExpirationDate] = useState("");
+
+  const [nonCompeteStatusHelper, setNonCompeteStatusHelper] = useState(
```

Comment:
Should we have helper states for all the fields? & error handling too?

This also leads into the question of which fields are required and which aren't. We may want to add a Required* to the labels of certain fields.